quetz/tasks/common.py

```diff
@@ -14,6 +14,11 @@
 def assert_channel_action(action, channel):
     if action == ChannelActionEnum.synchronize:
         action_allowed = assertions.can_channel_synchronize(channel)
+    elif action == ChannelActionEnum.validate_packages:
+        # TODO!!!
+        action_allowed = True
+    elif action == ChannelActionEnum.generate_indexes:
+        action_allowed = True
```

Comment:
I guess we don't want everyone to trigger these actions, it would be an easy way to DoS the server

Reply:
I think here we're only testing if the channel itself can do that (mirror vs real channel etc).